fapolicy_analyzer/ui/subject_list.py

```diff
@@ -107,23 +107,36 @@ def _columns(self):
 
     def _trust_markup(self, subject):
         u_str = "U"
-        at_str = '<span color=\"red\"><b>AT</b></span>'
-        st_str = '<span color=\"red\"><b>ST</b></span>'
+        at_str = "AT"
+        st_str = "ST"
+
+        status = (
+            dbtrust.status.lower()
+            if (dbtrust := self.__find_db_trust(subject))
+            else None
+        )
```

Comment:
You shouldn't need to do this any more to get the status. #363 added the `trust_status` property to the `subject` and `object`. At some point we should go back and update the older code that is doing it this way too...when time permits.